robot_smach_states/src/robot_smach_states/human_interaction/answer_questions.py

```diff
@@ -187,9 +111,8 @@ def answer_placement_location(action):
         return 'I dont know that object'
 
 
-def answer_count_placement(action):
+def answer_count_placement(action, common_knowledge):
```

Comment:
Code was removed  by @jlunenburg in this function because it was unreachable (always returned before the second if/else conditional). However functionality here might not be completely as it was supposed to be? -->Check!